aws-logs-metricfilter/aws-logs-metricfilter.json

```diff
@@ -10,6 +10,29 @@
   "description": "Specifies a metric filter that describes how CloudWatch Logs extracts information from logs and transforms it into Amazon CloudWatch metrics.",
   "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-logs.git",
   "definitions": {
+    "Dimension": {
+      "description": "the key-value pairs that further define a metric.",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "Key": {
+          "type": "string",
+          "description": "The key of the dimension. Maximum length of 255.",
+          "minLength": 1,
+          "maxLength": 255
```

Comment:
Needs to be changed to 250 - see [here](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Embedded_Metric_Format_Specification.html#CloudWatch_Embedded_Metric_Format_Specification_structure_example)